### ISSUE TITLE
DryRunner example not working

### ISSUE THREAD

**Issue by opensean** [kedro-org/kedro#2607]
## Description

There is a typo in DryRunner example implementation of the AbstractRunner class here https://docs.kedro.org/en/stable/nodes_and_pipelines/run_a_pipeline.html#custom-runners.

Remove the "\n" argument here https://github.com/kedro-org/kedro/blob/41f03d903cda4d4b4f61b01091f494b7b0817e79/docs/source/nodes_and_pipelines/run_a_pipeline.md?plain=1#L114 and the example works as expected.

Also the following imports are redundant and can be removed:
- https://github.com/kedro-org/kedro/blob/41f03d903cda4d4b4f61b01091f494b7b0817e79/docs/source/nodes_and_pipelines/run_a_pipeline.md?plain=1#L66
- https://github.com/kedro-org/kedro/blob/41f03d903cda4d4b4f61b01091f494b7b0817e79/docs/source/nodes_and_pipelines/run_a_pipeline.md?plain=1#L67
- https://github.com/kedro-org/kedro/blob/41f03d903cda4d4b4f61b01091f494b7b0817e79/docs/source/nodes_and_pipelines/run_a_pipeline.md?plain=1#L68

## Context
Trying to run the DryRunner example to become familiar with custom runners.

## Steps to Reproduce
1. follow the steps in the documentation to run the DryRunner example.

## Expected Result
Running the example as documented should produce the expected output.

## Actual Result
Example does not run as documented resulting in a :

```
TypeError: not all arguments converted during string formatting
```

**Comment by noklam** [comment]
@opensean Hey! Would you be open to create a PR to fix this?

**Comment by astrojuanlu** [comment]
Closed in #2827
